### PR TITLE
Add new FAQ page and link it to the top-level menu

### PR DESCRIPTION
**Pull request by sbesson**
Discussed at the last IDR meeting, this change ports the initial set of FAQs into a page to be deployed in the upcoming `prod84` release.

Deployed at https://snoopycrimecop.github.io/idr.openmicroscopy.org/faq/ for review:

- content: the Q&As have been going through several review iterations so I would focus on obvious typos
- layout: Q&As have been formatted as YAML to simplify maintenance and readability. The `h2` class used elsewhere in the static website is used as the way to format the questions but this is open to alternate questions
- URL: as it stands this would deploy the FAQ under idr.openmicroscopy.org/about/faq in production with idr.openmicroscopy.org/about/faq.html redirecting

### CODE REVIEW

File: faq/index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      Bio-Formats supports over 150 proprietary and open file formats (see the
+      <a href="https://docs.openmicroscopy.org/bio-formats/latest/supported-formats.html">full list</a>).
+      At a minimum, your data should be readable by an up-to-date version of
+      Bio-Formats e.g. using the <a href="https://docs.openmicroscopy.org/bio-formats/latest/users/comlinetools/display.html">showinf utility</a>
+      or the Image/Fiji plugin.
+  - question: What are the preferred formats for submission?
+    answer: >
+      We strongly encourage imaging data deposition using open file formats
+      such as OME-TIFF. Where possible the original raw data should be
+      submitted, though you can also submit analysed or processed data if you
```
Comment (mtbc) [inline faq/index.html:19]:
```suggestion
      submitted, though you can also submit analyzed or processed data if you
```
(if we are still doing US spelling for OME docs)

File: faq/index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      think it is more usable.
+  - question: Is there a size limit for data deposition?
+    answer: >
+      There is no size limit to the data that can be published in IDR. However
+      for very large depositions (>10TB), please contact us as early as
+      possible as special arrangements may be needed for data transfer.
+  - question: >
+      Is it possible to restrict access to the data for some period? For 
+      example, is it possible for data to embargoed until publication of the 
+      full to select an embargo?
```
Comment (mtbc) [inline faq/index.html:29]: Using "full" as a noun here reads oddly to me.

File: faq/index.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+      peer-reviewed paper. <a href="mailto:idr@openmicroscopy.org">Contact the
+      IDR team</a> for more info.
+  - question: Is it possible to schedule a release date? 
+    answer: >
+      Scheduling a release date is possible, but will often be provisional.
+      However, advance notice is required as we will only schedule a release
+      date once all data has been submitted, validated, and all our data
+      integrity and consistency checks are satisfied.  The amount of work and
+      time this requires and our ability to honor an agreed publication date
+      depends on the complexity of the datasets, and in many cases, how
```
Comment (mtbc) [inline faq/index.html:48]:
```suggestion
      depends on the complexity of the datasets and, in many cases, how
```

File: faq/index.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+      The repository is hosted on EMBL-EBI infrastructure and as such a large
+      portion of the security policy comes from EBI’s own policies. The IDR
+      team has its own internal security policy which includes the protection
+      of sensitive data such as embargoed submissions.
+  - question: >
+      How does IDR handle private health identifiers (PHI) or personal
+      identifying information (PII) that may be included in the data or
+      metadata?
+    answer: >
+      IDR publishes fully anonymized data. We request that the submitters
```
Comment (mtbc) [inline faq/index.html:90]:
```suggestion
      IDR publishes fully anonymized data. We require that the submitters
```

**Comment by joshmoore** [comment]
* "Q&As have been formatted as YAML": 💯 
 * "The h2 class used elsewhere in the static website is used as the way to format the questions": for a start no objections. I imagine eventually we might want an accordion effect.
 * If the reason for having FAQ on the bar is that about/ is too cluttered, I could see addressing that (soonish).

**Comment by sbesson** [comment]
Agreed on the accordion mid-term. Maybe some FAQ categories eventually.

Yes i was slightly concerned about burying the FAQ entry under the About menu. I dont feel particularly strong about this so happy to move under About and take the time to review the menu bar separately.

**Pull request merged**